### DISCUSSION TITLE
Average value of RSSI and Last RSSI not changing.

### DISCUSSION THREAD

**Discussion by HimanshuTembhe** [openthread/openthread#10352] [Q&A] [answered]
@jwhui as seen in the images , the distance between node 1 and node 2 is been changed but values of RSSI are not changing , in the router table as well as neighbor linkquality. value of LQ is also not changed , it remains 0 or 3 . Can we get different values for RSSI and Link quality now ? Or can we expect in new update of OTNS ?
![pic4](https://github.com/openthread/openthread/assets/99601153/872bffc5-479b-465b-8c55-4cd02973e5e4)
![pic5](https://github.com/openthread/openthread/assets/99601153/f94dedc2-fbb7-474a-ad8e-3a5ea83cad21)

**Comment by jwhui** [comment]
Existing OTNS implements unit disk radio model and does not compute RSSI based on distance between nodes.

@EskoDijk is looking into more complex radio models in a new version of OTNS.

**Comment by HimanshuTembhe** [comment]
Is there any command which can show change in values according to distance in current version of OTNS ?

**Reply by jwhui** [reply]
As I mentioned above, the existing OTNS implements unit disk radio model and does not change RSSI based on distance.

**Comment by EskoDijk** [comment, marked as answer]
@HimanshuTembhe The newer version that Jonathan mentioned is currently hosted here: https://github.com/EskoDijk/ot-ns/ 
In case you'd like to try it. The default radiomodel various RSSI based on distance and on random time/position based fluctuations. I'm now preparing it for contributing back to the openthread/ot-ns repo.

**Reply by EskoDijk** [reply]
Meanwhile the OTNS2 updates are now merged back into OTNS main: [https://github.com/openthread/ot-ns/](https://github.com/openthread/ot-ns/)
